src/net.rs

```diff
@@ -58,15 +80,18 @@ impl<'de> Deserialize<'de> for SignerID {
 #[derive(PartialEq, Debug, Serialize, Deserialize)]
 pub enum MessageType {
     Candidateblock(Block),
-    Signature(Signature),
     Completedblock(Block),
+    Nodevss(VerifiableSS, FE),
+    Blockvss(Block, VerifiableSS, FE),
+    Blocksig(Block, FE, FE),
```

Comment:
It seems the block in Blocksig message is not used. 

I suggest following change.
Use block hash instead of block and check the block hash is same with block data in node state when the node receives Blocksig message.

Reply:
I'll change fields of message blockvss and blocksig like this,

```
Blockvss(BlockHash, VerifiableSS, FE),
Blocksig(BlockHash, FE, FE),
```

And , I think members should store candidate_block when they receive it.

Reply:
Fix at 1daceaa301ea2bdb0bfc54d34928dbdbde376619